lib/main_common.pm

```diff
@@ -778,4 +791,15 @@ sub load_systemd_patches_tests {
     loadtest 'console/systemd_testsuite';
 }
 
+sub load_create_hdd_tests {
+    return unless get_var('INSTALLONLY');
+    # temporary adding test modules which applies hacks for missing parts in sle15
+    loadtest 'console/sle15_workarounds' if is_sle && sle_version_at_least('15');
+    loadtest 'console/hostname' unless is_bridged_networking;
```

Comment:
is it intended that we will run hostname in opensuse too ? as there is no is_sle check added

Reply:
I see no reason not to run it except for the additional time it takes but that is negligible compared to force_cron_run and also when we invest the time for the SLE tests why shouldn't the same apply for openSUSE?

Reply:
don't want to argue about idea , just wanted to double check that it was your intention - if answer is yes all is fine :)